SQL/Archive/17.1/2016-11-21-CreateUserSitesRelTable.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE IF EXISTS `user_psc_rel`;
+CREATE TABLE `user_psc_rel` (
+  `UserID` int(10) unsigned NOT NULL default '0',
+  `CenterID` tinyint(2) unsigned default NULL,
```

Comment:
On mysql 5.7, according to ERROR 1171 (42000): All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead